scripts/lib/cleanup.sh

```diff
@@ -6,19 +6,21 @@ function check_path_cleanup(){
     do
         export NODE=$node
         envsubst '$NODE' < ${DIR}/test/check-cleanup-pod.yaml > ${DIR}/test/check-cleanup-pod-$node.yaml
-        kubectl apply -f ${DIR}/test/check-cleanup-pod-$node.yaml
+        kubectl apply -f ${DIR}/test/check-cleanup-pod-$node.yaml -n default
         rm -rf ${DIR}/test/check-cleanup-pod-$node.yaml
     done
     sleep 20
 
     for node in $worker_nodes
     do
-        if [[ $(kubectl get pods $node -ojsonpath="{.status.phase}") == "Failed" ]]; then
-            echo "BPF files not cleaned up on $node.. $(kubectl logs $node)"
-            exit 1
+        if [[ $(kubectl get pods -n default $node -ojsonpath="{.status.phase}") == "Failed" ]]; then
```

Comment:
Trying to understand this, are we checking the pod status to determine if the BPF files are cleaned up? Am I missing anything? We should check on the node in /sys/fs/bpf and see if there are any leaked maps/progs or we could also just dump the NA CLI to check. But after cleanup we should have just 2 maps i.e, global map - ringbuffer and conntrack map. There will be 2 additional maps which is loaded by default which we can ignore.

Reply:
Yes, the pod returns a failure if either the maps/programs are not cleaned up (except the two global maps). The pod mounts the host file system and runs the check if there are any leaked resources on every instance.